NPB/Makefile

```diff
@@ -0,0 +1,71 @@
+SHELL=/bin/sh
+CLASS=W
+SFILE=config/suite.def
+
+default: header
+	@ $(SHELL) sys/print_instructions
+
+BT: bt
+bt: header
+	cd BT; $(MAKE) CLASS=$(CLASS)
+		       
+SP: sp		       
+sp: header	       
+	cd SP; $(MAKE) CLASS=$(CLASS)
+		       
+LU: lu		       
+lu: header	       
+	cd LU; $(MAKE) CLASS=$(CLASS)
+		       
+MG: mg		       
+mg: header	       
+	cd MG; $(MAKE) CLASS=$(CLASS)
+		       
+FT: ft		       
+ft: header	       
+	cd FT; $(MAKE) CLASS=$(CLASS)
+		       
+IS: is		       
+is: header	       
+	cd IS; $(MAKE) CLASS=$(CLASS)
+		       
+CG: cg		       
+cg: header	       
+	cd CG; $(MAKE) CLASS=$(CLASS)
+		       
+EP: ep		       
+ep: header	       
+	cd EP; $(MAKE) CLASS=$(CLASS)
+DC: dc
+dc: header	       
+	cd DC; $(MAKE) CLASS=$(CLASS)
+
+# Awk script courtesy cmg@cray.com
+suite:
+	@ awk '{ if ($$1 !~ /^#/ &&  NF > 0)                              \
+	printf "make %s CLASS=%s\n", $$1, $$2 }' $(SFILE)  \
+	| $(SHELL)
+
+
+# It would be nice to make clean in each subdirectory (the targets
+# are defined) but on a really clean system this will won't work
+# because those makefiles need config/make.def
+clean:
+	- rm -f core 
+	- rm -f *~ */core */*~ */*.o */npbparams.hpp */*.obj */*.exe
+	- rm -f sys/setparams sys/makesuite sys/setparams.hpp
+
+cleanall: clean
+	- rm -r bin/*
+
+veryclean: clean
+	- rm config/make.def config/suite.def Part*
```

Comment:
Why do you delete a file that is in git?

Reply:
The `veryclean` option was always part of the NPB I think, for people who want to delete even the config files... I don't use it, if you want, we can delete it and have only `clean` and `cleanall`.